disputes/mainnet/1-app-mining-redemptions/evidences/1.md

```diff
@@ -1,5 +1,5 @@
-1Hive’s Redemptions app allows Aragon organizations to grant their token holders the right to redeem tokens in exchange for a proportional share of the organizations treasury assets.

```

Comment:
Maybe not a big deal, but does this file still hash to the same CID?

Reply:
I wouldn't expect this to change cause I didn't change the file content